modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -323,12 +323,12 @@ include::devguide:example$nodejs/capella_connection_crud.ts[tag=json,indent=0]
 
 === Insert (Create) and Upsert
 
-`insert` and `upsert` will both create a new document.
-The difference between the two is that if a document with that key already exists, the `insert` operation will fail,
+`insert` and `upsert` both create a new document.
+The difference is, if a document with a key exists already, the `insert` operation fails 
 // throwing `DocumentExistsException` --
-while the `upsert` operation will succeed, replacing the content.
+and the `upsert` operation replaces the content and succeeds.
```

Comment:
The key here is it succeeds,
replacing the content is the literal side effect of that success, so that clause should follow for maximum clarity in reading the whole sentence.